.github/workflows/health_base.yaml

```diff
@@ -0,0 +1,155 @@
+# A CI configuration to check PR health.
+
+name: Health:Base
+
+# Callers of this workflow should use it as follows:
+#
+# name: Health
+# on:
+#   pull_request:
+#     branches: [ main ]
+#     types: [opened, synchronize, reopened, labeled, unlabeled]
+# jobs:
+#   health:
+#     uses: dart-lang/ecosystem/.github/workflows/health.yaml@main
```

Comment:
Should this be `health_base.yaml` instead?

Which of `health.yaml` and `health_base.yaml` are going to be used from a repositories `.github/workflows/health.yaml`? Both? Or one of the two? Please document in both how they are related, how to chose which one.

Both or just `health.yaml`? Since this documenting `health.yaml` in `health_base.yaml` now?